extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -21,10 +21,11 @@
           "title": "Interaction Type",
           "type": "string",
           "description": "Specify action made by user on a message.",
-          "enum": ["click", "open", "action"],
+          "enum": ["click", "open", "unsubscribe"],
           "meta:enum": {
             "click": "Link in a message is clicked by the user.",
-            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application."
+            "open": "Either Message is open by the user, or in case of push, clicking the message opened the mobile application.",
```

Comment:
@guayang-adobe - Please be aware that meta:enum values are displayed as drop downs  in ansible UI mainly used for segmentation or filtering. short description like below is recommended for them to be displayed properly in UI. 
“click”: “Message link clicked.“,
“open”: “Message opened.“,
“unsubscribe”: “Unsubscribed from push messages”